### PR TITLE
[Report Page] Add AMR view.

### PR DESCRIPTION
**Pull request by tfrcarvalho**
# Description

* Link AMR tab to AMR table view
* Add new endpoint to load AMR information

# Tests

* Verify that clicking the AMR view shows the AMR table.

### CODE REVIEW

File: app/assets/src/components/views/SampleViewV2/SampleViewV2.jsx [hunk 1]
```diff
@@ -22,6 +22,7 @@ import {
   getSampleReportData,
   getSamples,
 } from "~/api";
+import { getAmrData } from "~/api/amr";
 import { UserContext } from "~/components/common/UserContext";
 import { AMR_TABLE_FEATURE } from "~/components/utils/features";
```
Comment (MarkAZhang) [inline app/assets/src/components/views/SampleViewV2/SampleViewV2.jsx]: `/components/utils/features` is great! I didn't know about that.

File: app/controllers/samples_controller.rb [hunk 2]
```diff
@@ -49,6 +49,9 @@ class SamplesController < ApplicationController
   before_action only: :show_v2 do
     allowed_feature_required("report_v2")
   end
+  before_action only: :amr do
```
Comment (MarkAZhang) [inline app/controllers/samples_controller.rb]: Add an rspec test for user authorization.

File: app/controllers/samples_controller.rb [hunk 3]
```diff
@@ -779,6 +782,18 @@ def report_v2
     render json: PipelineReportService.call(pipeline_run.id, background_id)
   end
 
+  def amr
```
Comment (MarkAZhang) [inline app/controllers/samples_controller.rb]: Worth adding an rspec test for this?

Comment (tfrcarvalho) [reply app/controllers/samples_controller.rb]: Will not add at this moment.
Did not change the code that generates it, and should be eventually reviewer.

File: app/controllers/samples_controller.rb (hunk 2 above)
Comment (tfrcarvalho) [reply app/controllers/samples_controller.rb]: Added.

**Pull request merged**